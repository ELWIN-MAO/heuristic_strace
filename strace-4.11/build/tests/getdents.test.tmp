
mym_waitpid=101851

mym_waitpid=101851

mym_waitpid=101851

syscall_enter_mym_scno=59  sys_name=execve

mym_waitpid=101851

mym_waitpid=101851

syscall_exit_mym_scno=59  sys_name=execve

mym_waitpid=101851

syscall_enter_mym_scno=12  sys_name=brk

mym_waitpid=101851

syscall_exit_mym_scno=12  sys_name=brk

mym_waitpid=101851

syscall_enter_mym_scno=21  sys_name=access

mym_waitpid=101851

syscall_exit_mym_scno=21  sys_name=access

mym_waitpid=101851

syscall_enter_mym_scno=9  sys_name=mmap

mym_waitpid=101851

syscall_exit_mym_scno=9  sys_name=mmap

mym_waitpid=101851

syscall_enter_mym_scno=21  sys_name=access

mym_waitpid=101851

syscall_exit_mym_scno=21  sys_name=access

mym_waitpid=101851

syscall_enter_mym_scno=2  sys_name=open

mym_waitpid=101851

syscall_exit_mym_scno=2  sys_name=open

mym_waitpid=101851

syscall_enter_mym_scno=5  sys_name=fstat

mym_waitpid=101851

syscall_exit_mym_scno=5  sys_name=fstat

mym_waitpid=101851

syscall_enter_mym_scno=9  sys_name=mmap

mym_waitpid=101851

syscall_exit_mym_scno=9  sys_name=mmap

mym_waitpid=101851

syscall_enter_mym_scno=3  sys_name=close

mym_waitpid=101851

syscall_exit_mym_scno=3  sys_name=close

mym_waitpid=101851

syscall_enter_mym_scno=21  sys_name=access

mym_waitpid=101851

syscall_exit_mym_scno=21  sys_name=access

mym_waitpid=101851

syscall_enter_mym_scno=2  sys_name=open

mym_waitpid=101851

syscall_exit_mym_scno=2  sys_name=open

mym_waitpid=101851

syscall_enter_mym_scno=0  sys_name=read

mym_waitpid=101851

syscall_exit_mym_scno=0  sys_name=read

mym_waitpid=101851

syscall_enter_mym_scno=5  sys_name=fstat

mym_waitpid=101851

syscall_exit_mym_scno=5  sys_name=fstat

mym_waitpid=101851

syscall_enter_mym_scno=9  sys_name=mmap

mym_waitpid=101851

syscall_exit_mym_scno=9  sys_name=mmap

mym_waitpid=101851

syscall_enter_mym_scno=10  sys_name=mprotect

mym_waitpid=101851

syscall_exit_mym_scno=10  sys_name=mprotect

mym_waitpid=101851

syscall_enter_mym_scno=9  sys_name=mmap

mym_waitpid=101851

syscall_exit_mym_scno=9  sys_name=mmap

mym_waitpid=101851

syscall_enter_mym_scno=9  sys_name=mmap

mym_waitpid=101851

syscall_exit_mym_scno=9  sys_name=mmap

mym_waitpid=101851

syscall_enter_mym_scno=3  sys_name=close

mym_waitpid=101851

syscall_exit_mym_scno=3  sys_name=close

mym_waitpid=101851

syscall_enter_mym_scno=9  sys_name=mmap

mym_waitpid=101851

syscall_exit_mym_scno=9  sys_name=mmap

mym_waitpid=101851

syscall_enter_mym_scno=9  sys_name=mmap

mym_waitpid=101851

syscall_exit_mym_scno=9  sys_name=mmap

mym_waitpid=101851

syscall_enter_mym_scno=158  sys_name=arch_prctl

mym_waitpid=101851

syscall_exit_mym_scno=158  sys_name=arch_prctl

mym_waitpid=101851

syscall_enter_mym_scno=10  sys_name=mprotect

mym_waitpid=101851

syscall_exit_mym_scno=10  sys_name=mprotect

mym_waitpid=101851

syscall_enter_mym_scno=10  sys_name=mprotect

mym_waitpid=101851

syscall_exit_mym_scno=10  sys_name=mprotect

mym_waitpid=101851

syscall_enter_mym_scno=10  sys_name=mprotect

mym_waitpid=101851

syscall_exit_mym_scno=10  sys_name=mprotect

mym_waitpid=101851

syscall_enter_mym_scno=11  sys_name=munmap

mym_waitpid=101851

syscall_exit_mym_scno=11  sys_name=munmap

mym_waitpid=101851

syscall_enter_mym_scno=12  sys_name=brk

mym_waitpid=101851

syscall_exit_mym_scno=12  sys_name=brk

mym_waitpid=101851

syscall_enter_mym_scno=12  sys_name=brk

mym_waitpid=101851

syscall_exit_mym_scno=12  sys_name=brk

mym_waitpid=101851

syscall_enter_mym_scno=83  sys_name=mkdir

mym_waitpid=101851

syscall_exit_mym_scno=83  sys_name=mkdir

mym_waitpid=101851

syscall_enter_mym_scno=80  sys_name=chdir

mym_waitpid=101851

syscall_exit_mym_scno=80  sys_name=chdir

mym_waitpid=101851

syscall_enter_mym_scno=3  sys_name=close

mym_waitpid=101851

syscall_exit_mym_scno=3  sys_name=close

mym_waitpid=101851

syscall_enter_mym_scno=85  sys_name=creat

mym_waitpid=101851

syscall_exit_mym_scno=85  sys_name=creat

mym_waitpid=101851

syscall_enter_mym_scno=3  sys_name=close

mym_waitpid=101851

syscall_exit_mym_scno=3  sys_name=close

mym_waitpid=101851

syscall_enter_mym_scno=2  sys_name=open

mym_waitpid=101851

syscall_exit_mym_scno=2  sys_name=open

mym_waitpid=101851

syscall_enter_mym_scno=78  sys_name=getdents
getdents(0, 
mym_waitpid=101851

syscall_exit_mym_scno=78  sys_name=getdents
[{d_ino=2491382, d_off=3539870932030788430, d_reclen=24, d_name=".", d_type=DT_DIR}, {d_ino=2490480, d_off=4576621732010379221, d_reclen=24, d_name="..", d_type=DT_DIR}, {d_ino=2491383, d_off=9223372036854775807, d_reclen=280, d_name="A\nA\nA\nA\nA\nA\nA\nA\nA\nA\nA\nA\nA\nA\nA\nA\nA\nA\nA\nA\nA\nA\nA\nA\nA\nA\nA\nA\nA\nA\nA\nA\nA\nA\nA\nA\nA\nA\nA\nA\nA\nA\nA\nA\nA\nA\nA\nA\nA\nA\nA\nA\nA\nA\nA\nA\nA\nA\nA\nA\nA\nA\nA\nA\nA\nA\nA\nA\nA\nA\nA\nA\nA\nA\nA\nA\nA\nA\nA\nA\nA\nA\nA\nA\nA\nA\nA\nA\nA\nA\nA\nA\nA\nA\nA\nA\nA\nA\nA\nA\nA\nA\nA\nA\nA\nA\nA\nA\nA\nA\nA\nA\nA\nA\nA\nA\nA\nA\nA\nA\nA\nA\nA\nA\nA\nA\nA\nZ", d_type=DT_REG}], 8192
mym_scall_rval=0x148
) = 328

mym_waitpid=101851

syscall_enter_mym_scno=5  sys_name=fstat

mym_waitpid=101851

syscall_exit_mym_scno=5  sys_name=fstat

mym_waitpid=101851

syscall_enter_mym_scno=78  sys_name=getdents
getdents(0, 
mym_waitpid=101851

syscall_exit_mym_scno=78  sys_name=getdents
[], 8192
mym_scall_rval=0
) = 0

mym_waitpid=101851

syscall_enter_mym_scno=87  sys_name=unlink

mym_waitpid=101851

syscall_exit_mym_scno=87  sys_name=unlink

mym_waitpid=101851

syscall_enter_mym_scno=80  sys_name=chdir

mym_waitpid=101851

syscall_exit_mym_scno=80  sys_name=chdir

mym_waitpid=101851

syscall_enter_mym_scno=84  sys_name=rmdir

mym_waitpid=101851

syscall_exit_mym_scno=84  sys_name=rmdir

mym_waitpid=101851

syscall_enter_mym_scno=1  sys_name=write

mym_waitpid=101851

syscall_exit_mym_scno=1  sys_name=write

mym_waitpid=101851

syscall_enter_mym_scno=231  sys_name=exit_group

mym_waitpid=101851
+++ exited with 0 +++
