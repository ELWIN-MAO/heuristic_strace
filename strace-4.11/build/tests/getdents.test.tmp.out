getdents(0, [{d_ino=2491382, d_off=3539870932030788430, d_reclen=24, d_name=".", d_type=DT_DIR}, {d_ino=2490480, d_off=4576621732010379221, d_reclen=24, d_name="..", d_type=DT_DIR}, {d_ino=2491383, d_off=9223372036854775807, d_reclen=280, d_name="A\nA\nA\nA\nA\nA\nA\nA\nA\nA\nA\nA\nA\nA\nA\nA\nA\nA\nA\nA\nA\nA\nA\nA\nA\nA\nA\nA\nA\nA\nA\nA\nA\nA\nA\nA\nA\nA\nA\nA\nA\nA\nA\nA\nA\nA\nA\nA\nA\nA\nA\nA\nA\nA\nA\nA\nA\nA\nA\nA\nA\nA\nA\nA\nA\nA\nA\nA\nA\nA\nA\nA\nA\nA\nA\nA\nA\nA\nA\nA\nA\nA\nA\nA\nA\nA\nA\nA\nA\nA\nA\nA\nA\nA\nA\nA\nA\nA\nA\nA\nA\nA\nA\nA\nA\nA\nA\nA\nA\nA\nA\nA\nA\nA\nA\nA\nA\nA\nA\nA\nA\nA\nA\nA\nA\nA\nA\nZ", d_type=DT_REG}], 8192) = 328
getdents(0, [], 8192) = 0
+++ exited with 0 +++
lo IP Address 127.0.0.1
ens33 IP Address 192.168.159.137
IP Address 127.0.0.1
IP Address 192.168.159.137
